src/scenes/Wallet/WalletFund.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { SafeOnRampKit, SafeOnRampProviderType } from '@safe-global/onramp-kit'
+
+export interface WalletFundProps {
+  address: string;
+};
+
+function WalletFund() {
+
+  const [address, setAddress] = useState<string>(localStorage.getItem('safeAddress')||'');
+
+  function handleAddressChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setAddress(event.target.value);
+  }
+
+  const fundWallet = async function() {
```

Comment:
You use different code style between this fn and the above one. I would use the `const fn = () => {}` as is the typical one in our repos

Reply:
Good catch. Fixed.